### ISSUE TITLE
Error with Rails 6 when getting layout name for a template

### ISSUE THREAD

**Issue by djmb** [DataDog/dd-trace-rb#857]
I noticed that `rails.render_template` wasn't recording the template name as a tag.

Digging in it looks like the issue is here - https://github.com/DataDog/dd-trace-rb/blob/346634da8c40803fecad891c2cf524751a1dd78b/lib/ddtrace/contrib/action_view/instrumentation/template_renderer.rb#L113

Under Rails 6.0.1, I'm getting this error:
```
[1] pry(#<ActionView::TemplateRenderer>)> layout_name.try(:[], 'virtual_path')
NoMethodError: Could not render layout: undefined method `find_all' for "virtual_path":String
from <redacted>/.gem/ruby/2.6.3/gems/actionview-6.0.1/lib/action_view/layouts.rb:418:in `rescue in _default_layout'
Caused by NoMethodError: undefined method `find_all' for "virtual_path":String
from <redacted>/.gem/ruby/2.6.3/gems/actionview-6.0.1/lib/action_view/layouts.rb:327:in `_layout'
```

**Comment by delner** [comment]
Hmmm, that doesn't look right. I'll set this up for triage. Thanks for the report @djmb! We'll take a look at this and get back to you.

**Comment by marcotc** [comment]
@djmb, thank you for your help with this issue.

We've just released a fix for it in version [0.30.0](https://github.com/DataDog/dd-trace-rb/releases/tag/v0.30.0).

Let us know if the problem still persists with the new version.

**Comment by djmb** [comment]
Hi @marcotc - thanks for this. I've upgraded to 0.30.0 and can confirm the issue is fixed for us.
